Add blog post about StrimziCon 2024 (#408)
strimzi/strimzi.github.io: _posts/2024-01-29-strimzicon2024-announcement.md

---
layout: post
title:  "Welcome to the StrimziCon 2024!"
date: 2024-01-29
author: paolo_patierno
---

We are very pleased to announce the first StrimziCon coming this year!

> This is going to be the first conference focused on our beloved Strimzi project and it will be "virtual".

Reviewer: Maybeyou can try to _improve the SEO_ and add also Apache Kafka into it?

> It is organized and driven by the project maintainers together with CNCF.

Reviewer 2: ```suggestion
It is organized and driven by the project maintainers in collaboration with the CNCF.

```

> It is organized and driven by the project maintainers together with CNCF.

Reviewer: Should we use the full name and link for CNCF?

<!--more-->

![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)

> The StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.

Reviewer 2: ```suggestion
StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.

```

> This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.

Reviewer 2: ```suggestion
This event will bring together the Strimzi community, where you can learn from experts and users about use cases, best practices, and their experiences using Strimzi in their organizations.

```

Author: It's the official description already fixed by different people I would stick with that.

> If you are working with cloud-native technologies, and you are looking for how to deal with data in motion in a cloud-based environment, don’t miss the opportunity.

Reviewer 2: ```suggestion
If you are working with cloud-native technologies and looking for ways to manage data in motion in a cloud-based environment, don’t miss this opportunity.

```

Author: Ditto as above.

> It will take place on **Wednesday, May 22nd** (afternoon CEST) with two parallel tracks on the following themes:

Reviewer 2: ```suggestion
It will take place on **Wednesday, May 22nd** (afternoon CEST) and will feature two parallel tracks on the following themes:

```

* Features, How Strimzi Works.
* Operations, Using Strimzi with other projects.

> The Call for Speakers is opened and it's waiting for you sending your proposals to talk about:

Reviewer: ```suggestion
The Call for Speakers is open and it's waiting for you to send your proposals to talk about:
```

* **Core Strimzi**: Deep dive into a specific component or feature.
* **Case Studies**: Overview of a use case or review of a real life Strimzi deployment, difficulties, gotcha, main takeaways.
* **Integrations**: How Strimzi can be used together with other CNCF projects (i.e. OpenTelemetry, KEDA, …).
* **Operations**: What it takes to run Strimzi and maintain cluster in production.

It will close on March 10th.

All the submitted sessions will be evaluated by the StrimziCon committee made up of people from different companies which already use Strimzi in production as adopters.

> The committee members will leverage their experience and knowledge on the project as users and contributors in order to select the best proposals and make an awesome agenda.

Reviewer: Should we list them here?

You can find more details on the official [page](https://community.cncf.io/events/details/cncf-virtual-project-events-2024-hosted-by-cncf-presents-strimzicon-2024-virtual/) as part of the CNCF events and don't forget to submit your proposal [here](https://sessionize.com/strimzicon-2024/).

If you don't want to present, we hope you will join us anyway as an attendee.

> It's virtual, it's free, it's going to be epic, all reasons to be together with the Strimzi community at the StrimziCon!

Reviewer 2: Is this a slogan? :D If so it's very very nice!

---
layout: post
title:  "Welcome to the StrimziCon 2024!"
date: 2024-01-26
author: paolo_patierno
---

> We are very pleased to announce the first StrimziCon coming this year!

Reviewer 3: ```suggestion
We are very pleased to announce the inaugural StrimziCon, scheduled later this year!
```

Author: Why is "inaugural" better than "first"?

Reviewer 3: I thought inaugural sounded more significant and like something is being established. Just a suggestion

This is going to be the first conference focused on our beloved Strimzi and Apache Kafka projects and it will be "virtual".
It is organized and driven by the project maintainers in collaboration with the [Cloud Native Computing Foundation](https://www.cncf.io/).

<!--more-->

![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)

StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.
This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.
If you are working with cloud-native technologies, and you are looking for how to deal with data in motion in a cloud-based environment, don’t miss the opportunity.

> It will take place on **Wednesday, May 22nd** (afternoon CEST) and will feature two parallel tracks on the following themes:

Reviewer 3: ```suggestion
The conference will take place on **Wednesday, May 22nd** (afternoon CEST), with two parallel tracks covering the following themes:
```

> * Features, How Strimzi Works.
> * Operations, Using Strimzi with other projects.

Reviewer 3: ```suggestion
* Track 1: How Strimzi works (features)
* Track 2: Using Strimzi with other projects (operations)
```

Author: At the beginning I had "Track X" as well but then I thought ... "what's the point to write Track X if the sentence before mentions parallel tracks? It's obvious these themes are related to track X and Y", or?

Reviewer 3: Just to be clearer when glancing at the info and to structure it a bit. again, just a suggestion

> The Call for Speakers is open and it's waiting for you to send your proposals to talk about:

Reviewer 3: ```suggestion
The call for **speakers** is now open, and we're eagerly awaiting your proposals to discuss:
```

* **Core Strimzi**: Deep dive into a specific component or feature.

> * **Case Studies**: Overview of a use case or review of a real life Strimzi deployment, difficulties, gotcha, main takeaways.

Reviewer 3: ```suggestion
* **Case Studies**: Provide an overview of a use case or review of a real life Strimzi deployment, highlighting difficulties, lessons learned, and key takeaways.
```

> * **Integrations**: How Strimzi can be used together with other CNCF projects (i.e. OpenTelemetry, KEDA, …).

Reviewer 3: ```suggestion
* **Integrations**: Explore how Strimzi integrates with other CNCF projects, such as OpenTelemetry and KEDA.
```

> * **Operations**: What it takes to run Strimzi and maintain cluster in production.

Reviewer 3: ```suggestion
* **Operations**: Share insights into what it takes to run Strimzi and maintain a cluster in production.
```

> It will close on March 10th.

Reviewer 3: ```suggestion
The submission deadline is March 10th.
```

All the submitted sessions will be evaluated by the StrimziCon committee made up of people from different companies which already use Strimzi in production as adopters.

> We'll have representative from (in alphabetical order) Apple, Atruvia, Axual, Decathlon, IBM, Red Hat, and SBB.

Reviewer 3: ```suggestion
We'll have representatives from Apple, Atruvia, Axual, Decathlon, IBM, Red Hat, and SBB.
```

The committee members will leverage their experience and knowledge on the project as users and contributors in order to select the best proposals and make an awesome agenda.

> You can find more details on the official [page](https://community.cncf.io/events/details/cncf-virtual-project-events-2024-hosted-by-cncf-presents-strimzicon-2024-virtual/) as part of the CNCF events and don't forget to submit your proposal [here](https://sessionize.com/strimzicon-2024/).

Reviewer 3: ```suggestion
You can find more details on the official [page](https://community.cncf.io/events/details/cncf-virtual-project-events-2024-hosted-by-cncf-presents-strimzicon-2024-virtual/) as part of the CNCF events. Don't forget to submit your proposal [here](https://sessionize.com/strimzicon-2024/).
```

> If you don't want to present, we hope you will join us anyway as an attendee.

Reviewer 3: ```suggestion
Even if you're not considering presenting, we hope you will still join us as an attendee.
```

---
layout: post
title:  "Welcome StrimziCon 2024!"
date: 2024-01-26
author: paolo_patierno
---

We are very pleased to announce the first StrimziCon, scheduled later this year!
This is going to be the first conference focused on our beloved Strimzi and Apache Kafka projects and it will be "virtual".
It is organized and driven by the project maintainers in collaboration with the [Cloud Native Computing Foundation](https://www.cncf.io/).

<!--more-->

![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)

> ## Why attend StrimziCon?

Reviewer: You should use `###` for these to not make them the same level as the main title.

StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.
This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.
If you are working with cloud-native technologies, and you are looking for how to deal with data in motion in a cloud-based environment, don’t miss the opportunity.

The conference will take place on <b>Wednesday, May 22nd</b> (afternoon CEST), with two parallel tracks covering the following themes:

* Features, How Strimzi Works (features)
* Operations, Using Strimzi with other projects (operations)

The call for <b>speakers</b> is now open, and we're eagerly awaiting your proposals to discuss:

* <b>Core Strimzi</b>: Deep dive into a specific component or feature.
* <b>Case Studies</b>: Provide an overview of a use case or review of a real life Strimzi deployment, highlighting difficulties, lessons learned, and key takeaways.
* <b>Integrations</b>: Explore how Strimzi integrates with other CNCF projects, such as OpenTelemetry, KEDA and more.

> * <b>Operations</b>: Share insights into what it takes to run Strimzi and maintain cluster in production.

Reviewer 4: ```suggestion
* <b>Operations</b>: Share insights into what it takes to run Strimzi and maintain clusters in production.
```

The submission deadline is March 10th.

### From the community to the community

> We encourage any community members to submit a talk: from a Strimzi deep dive to how you operate and use it in production, through integrations with other projects.

Reviewer 4: ```suggestion
We encourage any community members to submit a talk: from a Strimzi deep dive, to how you operate and use it in production, through integrations with other projects.
```
